test/unit/auth/auth-config.spec.ts

```diff
@@ -892,6 +941,43 @@ describe('OIDCConfig', () => {
       expect(() => OIDCConfig.validate(partialRequest, true)).not.to.throw();
     });
 
+    it('should throw on OAuth responseType contains invalid parameters', () => {
+      const invalidRequest = deepCopy(clientRequest) as any;
+      invalidRequest.responseType.token = true;
+      expect(() => OIDCConfig.validate(invalidRequest, true))
+        .to.throw('"token" is not a valid OAuthResponseType parameter.');
+    });
+
+    it('should not throw when exact one OAuth responseType is true', () => {
+      const validRequest = deepCopy(clientRequest) as any;
+      validRequest.responseType.code = false;
+      validRequest.responseType.idToken = true;
+      expect(() => OIDCConfig.validate(validRequest, true)).not.to.throw();
+    });
+
+    it('should throw on two OAuth responseTypes set to true', () => {
```

Comment:
Can we also test the scenarios where only one responseType is set?

```
responseType = { idToken: true }
responseType = { code: true }
```

I assume these are valid use cases.